tools/generic_rename_outputs.cwl

```diff
@@ -0,0 +1,32 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: generic_rename_outputs
+doc: "Simple tool to rename inputs"
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 2000
+    coresMin: 1
+
+baseCommand: []
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >-
+      ${
+          var out = "renamed/"
+          var cmd = "mkdir " + out + ";";
+          for (var i = 0; i < inputs.input_files.length; i++) {
+            cmd += "cp " + inputs.input_files[i].path + " " + out + inputs.rename_to[i] + ";"
```

Comment:
maybe put newlines between the `cp` commands for readability

Reply:
like at the end: `;\n` of each cmd iteration?

Reply:
Yeah